ArduPlane/quadplane.cpp

```diff
@@ -4114,23 +4114,16 @@ float QuadPlane::get_land_airspeed(void)
     if (qstate == QPOS_APPROACH ||
         plane.control_mode == &plane.mode_rtl) {
```

Comment:
this was supposed to be qrtl not rtl, it can never be in rtl when this function is called (existing bug)